com/user.cfc

```diff
@@ -701,4 +707,118 @@ component displayname="user" output="false" {
 
     }
 
+
+    public any function sendMfaCode(required numeric userID, boolean blnResend){
```

Comment:
Don't forget to declare all variables with "local.", if its not a global variable.